src/components/TodoList.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Todo } from "../model"
+import SingleTodo from './SingleTodo';
+import "./styles.css";
+
+interface Props{
+    inputs: Todo[]
+    setInputs: React.Dispatch<React.SetStateAction<Todo[]>>;
+  }
+
+  const TodoList: React.FC<Props> = ({inputs, setInputs}) => 
+    <div className="todos">
+        {inputs.map(todo => (
+            <SingleTodo todo={todo} key={todo.id}
+            inputs={inputs}
+            setInputs={setInputs}/>

```

Comment:
Też złe formatowanie :D Jak ogarniesz styled-components to polecimy z dodaniem lintera. To jest narzędzie które ci poprawia formatowanie z automatu.